.github/workflows/promote.yaml

```diff
@@ -0,0 +1,32 @@
+# reusable workflow triggered manually
+name: Promote charm to other tracks and channels
+
+on:
+  workflow_dispatch:
+    inputs:
+      destination-channel:
+        description: 'Destination Channel'
+        required: true
+      origin-channel:
+        description: 'Origin Channel'
+        required: true
+      charm-name:
+        description: 'Charm subdirectory name'
+        required: true
+
+jobs:
+  promote-charm:
+    name: Promote charm
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Release charm to channel
+        # TODO: use canonical/charming-actions/promote-charm?
+        uses: canonical/charming-actions/release-charm@2.6.2
```

Comment:
Transferred comment from https://github.com/canonical/kfp-operators/pull/655/files#r1920090360

> promote-charm looks like a new action that might work better here? not sure

From the README.md it looks like it's similar to release-charm except that it cover multi-base charms. Our team does not support that. I will confirm if that's the only difference.

Reply:
I confirmed that it is designed for multi-base charms, see [this PR description](https://github.com/canonical/charming-actions/pull/138). so it is not needed in our case for now. `relase-charm` works well for single base.